lib/matobj2.gd

```diff
@@ -1351,8 +1350,8 @@ DeclareOperation( "Matrix", [ IsList ]);
 ##  <Ref Attr="CompatibleVectorFilter" Label="for a matrix object"/> value
 ##  of <A>M</A> (provided that such a representation exists).
 ##  <P/>
-##  The idea is that there should be an efficient way to multiply <M>v</M>
-##  and <A>M</A>.
+##  The idea is that there should be an efficient way to 
+##  form the product <A>vM</A>.
```

Comment:
The `<A>` element is intended to be used for arguments of the function in question, which are declared in the `Arg` attribute. In this case, we have `Arg="M"`, thus `<A>M</A>` is allowed. On the other hand, `v` is a "free" variable, thus `<M>v</M>` is an appropriate syntax but `<A>vM</A>` is not, and also `<A>v</A>` would not make sense.

Reply:
My reason for changing "multiply v with M" to "product vM" was to remove any possibility that Mv might be understood by some readers.  I have only recently got it fixed in my mind that "vector" means "row vector".  Of course I should be more familiar with GAPDoc conventions.  Hopefully "<M>vM</M> is acceptable.

Reply:
I agree that your new text is better! On the technical level, I guess maybe `<M>v</M><A>M</A>` would be correct? @ThomasBreuer ?

Reply:
Done

Reply:
Yes, `<M>v</M><A>M</A>` is correct.